crates/completion/src/completions/postfix/format_like.rs

```diff
@@ -157,6 +158,11 @@ impl FormatStrParser {
                         inexpr_open_count -= 1;
                     }
                 }
+                (State::Expr, ':') if chars.peek().copied() == Some(':') => {
+                    // path seperator
+                    current_expr.push(chr);
```

Comment:
```suggestion
                    current_expr.push_str("::");
                    chars.next();
```

to avoid some panic bloat.

Reply:
👍